javascript/passwordToggle.js

```diff
@@ -0,0 +1,19 @@
+const togglePassword = document.querySelectorAll('.eye-btn')
+
+// passwordToggle(show and hide)
+const passwordToggle = (e) => {
+  if (e.target.classList.contains('off-eye-btn')) {
+    e.target.classList.remove('off-eye-btn')
+    e.target.previousElementSibling.setAttribute('type', 'password')
+  }
+  else {
+    e.target.classList.add('off-eye-btn')
+    e.target.previousElementSibling.setAttribute('type', 'text')
+  }
+}
```

Comment:
반복되는 클래스명이니 변수에 저장하고 쓰시면 좋겠네요.

```suggestion
const passwordToggle = (e) => {
  const encryptClass = 'off-eye-btn'; // 이름은 더 고민이 필요하겠네요.. 🥹
  
  if (e.target.classList.contains(encryptClass)) {
    e.target.classList.remove(encryptClass)
    e.target.previousElementSibling.setAttribute('type', 'password')
    return;
  }
  
  e.target.classList.add(encryptClass)
  e.target.previousElementSibling.setAttribute('type', 'text')
}
```

Reply:
넵!! 조금 더 반복되는 부분을 줄여보도록 하겠습니다.